File: Service/model/suiki/exercice.js
var db = require('./../../config/db');

var Schema = db.Schema;
/**
 * Table Exercice:
 *  - Nom : Nom de l'exercice
 *  - Description: Description de l'exercice
 *  - Lien : Lien youtube de l'exercice
 */
var exercice = new Schema({
    nom : 'String',
    description : 'String',
    lien : 'String'
});

module.exports = db.model('Exercice' , exercice);

File: Suiki/app/src/main/java/com/suiki/suiki/Model/BddModel/Exercice.java
package com.suiki.suiki.Model.BddModel;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by Massil on 11/01/2017.
 */

public class Exercice implements Serializable{
    @SerializedName("_id")
    public String id;
    @SerializedName("nom")
    public String nom;
    @SerializedName("description")
    public String description;
    @SerializedName("lien")
    public String lien;
}
